bin/pygrb/pycbc_pygrb_plot_injs_results

```diff
@@ -269,9 +269,25 @@ trig_data = ppu.extract_trig_properties(
     keys
 )
 
-background = list(trig_data['network/reweighted_snr'].values())
+# Max BestNR values in each trial: these are stored in a dictionary keyed
+# by slide_id, as arrays indexed by trial number
+background = {k: np.zeros(len(v)) for k,v in trial_dict.items()}
+for slide_id in slide_dict:
+    for j, trial in enumerate(trial_dict[slide_id]):
+        # True whenever the trigger is in the trial
+        trial_cut = (trial[0] <= trig_data[keys[0]][slide_id][:])\
```

Comment:
For readability, I suggest defining a temporary variable to hold the value of `trig_data[keys[0]][slide_id][:]` as the first thing into the outer loop, before entering the loop over `trial_dict[slide_id]`.